source/brailleTables.py

```diff
@@ -355,6 +364,9 @@ def listTables() -> list[BrailleTable]:
 addTable("grc-international-es.utb", _("Spanish for Greek text"))
 # Translators: The name of a braille table displayed in the
 # braille settings dialog.
+addTable("hbo.utb", _("Biblical Hebrew"))

```

Comment:
I think I'd rename this:
```suggestion
addTable("hbo.utb", _("Hebrew Biblical"))
```
The language should come before its use IMO.